api/controllers/userController.js

```diff
@@ -0,0 +1,20 @@
+const { userService } = require('../services')
+
+const signIn = async (req, res) => {
+  const { email, password } = req.body;
+  
+  if (!email || !password) {
+    const error = new Error('KEY_ERROR')
+    error.statusCode = 400;
+    throw error;
+  }
+
+  try {
+    const accessToken = await userService.signIn(email, password);
+    res.status(200).json({ accessToken });
+  } catch (error) {
+    res.status(error).json({message: "error"});
+  }
+}
```

Comment:
try 위치를 조정해 주면 좋을 것 같습니다.
```suggestion
const signIn = async (req, res) => {
  try {
    const { email, password } = req.body;
  
    if (!email || !password) {
      const error = new Error('KEY_ERROR')
      error.statusCode = 400;
      throw error;
    } 
    
    const accessToken = await userService.signIn(email, password);
    res.status(200).json({ accessToken });
  } catch (error) {
    res.status(error).json({message: "error"});
  }
}
```